packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -633,6 +634,21 @@ class DragTarget<T extends Object> extends StatefulWidget {
   ///
   /// The builder can build different widgets depending on what is being dragged
   /// into this drag target.
+  ///
+  /// [onWillAccept] or [onWillAcceptWithDetails] is called when a draggable
+  /// enters the target. If either return true, the data will appear in
+  /// `candidateData`, if they return false, the data will appear in
```

Comment:
Can be updated to :

/// [onWillAccept] or [onWillAcceptWithDetails] is called when a draggable enters the target. 
/// If true, then the data will appear in `candidateData` else in `rejectedData`.

Reply:
I agree, this way is more clear that one of those methods should be null. Though I would add a comma after `candidateData`:

```dart
/// [onWillAccept] or [onWillAcceptWithDetails] is called when a draggable enters the target. 
/// If true, then the data will appear in `candidateData`, else in `rejectedData`.
```